portal-web/test/functional/com/liferay/portalweb/macros/LDAP.macro

```diff
@@ -24,6 +24,23 @@ definition {
 			locator1 = "Button#CLOSE_MODAL");
 	}
 
+	@description = "Enable LDAP debug logs at localhost"
```

Comment:
Don't need to mention localhost, as we don't know the host context at this stage